### PR TITLE
Feature/hit the api

### PR DESCRIPTION
**Pull request by emmyobonyo**
## Tasks worked on.
- GET API implemented in the REFRESH button.
- POST API implemented.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+      Accept: 'application/json, text/plain, "/"',
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({ user: name, score }),
+  })
+    .then((res) => res.json());
+  inputName.value = '';
+  inputScore.value = '';
+});
+
```
Comment (ZawadiSumba66) [inline src/index.js:41]: - [ ] Kindly use an arrow function which is a modern syntax provided by the ES6 feature that will make your code look better and neater.

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,47 @@
 // import _ from 'lodash';
 import './style.css';
 
+const form = document.getElementById('form');
```
Comment (ZawadiSumba66) [inline src/index.js:4]: - [ ] Lets make our code neater by removing commented code form the code base

File: README.md [hunk 3]
```diff
@@ -7,31 +7,34 @@
...
 ## Clone This Repository
-```$ git clone https://github.com/emmyobonyo/leaderboard.git```
-
-```$ cd leaderboard```
+```
+$ git clone https://github.com/emmyobonyo/leaderboard.git
+$ cd leaderboard
+```
 
 ## Run Project
```
Comment (ZawadiSumba66) [inline README.md:28]: - [ ] Kindly include the `npm install` command to install the dependencies before the `npm start`

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi @emmyobonyo 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (emmyobonyo) [reply src/index.js:4]: -Comments Removed

File: README.md (hunk 3 above)
Comment (emmyobonyo) [reply README.md:28]: -```npm install``` added

File: src/index.js (hunk 1 above)
Comment (emmyobonyo) [reply src/index.js:41]: -After some consultation, the code being referred to was different from the one above, which I've managed to work on.

**Review by ZawadiSumba66 — APPROVED** [review]
Hi @emmyobonyo again, 
### STATUS: APPROVED 👍 
Great job implementing the changes requested. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**